trappy/plotter/ILinePlotGen.py

```diff
@@ -34,6 +34,18 @@
 from IPython.display import display, HTML
 
 
+def to_dygraph_colors(color_map):
```

Comment:
This function should probably lie in trappy/trappy/plotter/ColorMap.py. The functionality is quite cool and I think the ColorMap.py can be extened to have a set of predefined colors (a mapping of RGB values and the palette should be documented in pydocs. The latter being out of the scope of this pull request. I can probably do some work here.